src/js/node/child_process.ts

```diff
@@ -1359,9 +1360,17 @@ function nodeToBun(item, index) {
   if (typeof item === "number") {
     return item;
   }
+  if (item instanceof StreamModule.Readable) {
```

Comment:
instanceof can throw, lets not use it

https://github.com/oven-sh/bun/blob/e6954c440e03d27c3b56d13a7834c2b7dfbb3123/src/js/node/stream.ts#L822-L824

We can check for the existence of a `pipe` function and an `on` function

Reply:
I can prevent the throw with a `typeof item === 'object'` guard too

Reply:
not quite, instanceof is overridable